streaming/index.js

```diff
@@ -577,19 +585,23 @@ const startServer = async () => {
   /**
    * @param {any} req
    * @param {SystemMessageHandlers} eventHandlers
-   * @returns {function(object): void}
+   * @returns {SubscriptionListener}
    */
   const createSystemMessageListener = (req, eventHandlers) => {
```

Comment:
I have a separate commit that completely refactors this code, so for now I've just added a minor guard to ensure messages from ruby over redis to streaming are well-formed.